# bluez4: add compile patch fixing some usb declaration mismatch
# Author: Michael 'Mickey' Lauer <mickey@vanille-media.de>
# Date:   Thu Jan 29 16:52:21 2009 +0000
#
# Acquired from OpenEmbedded
Upstream-Status: Pending

Index: bluez-4.27/tools/dfutool.c
===================================================================
--- bluez-4.27.orig/tools/dfutool.c
+++ bluez-4.27/tools/dfutool.c
@@ -59,7 +59,7 @@
 #endif
 
 #ifdef NEED_USB_GET_BUSSES
-static inline struct usb_bus *usb_get_busses(void)
+inline struct usb_bus *usb_get_busses(void)
 {
 	return usb_busses;
 }
